### DISCUSSION TITLE
common treasure in towns

### DISCUSSION THREAD

**Discussion by Platon244** [TownyAdvanced/Towny#7559] [Q&A] [answered]
what to do if a player puts 10 coins in his city, then 10 coins are also added in another city

**Comment by LlmDl** [comment, marked as answer]
Your economy plugin probably doesn't handle Unicode characters, and views
all 4 letter Cyrillic-named towns as the same name.

Don't use essentials economy.

**Reply by Platon244** [reply]
ok what i must use?

**Reply by LlmDl** [reply]
You can use anything you like, not many will fail on Unicode characters. The wiki's FAQ has a list of plugins to not use.
